lib/turn.rb

```diff
@@ -1,29 +1,122 @@
-require 'colorize'
+# require 'colorize'
 
 class Turn
 
-    attr_reader :player_board,
-                :cpu_board,
-                :cpu_cruiser,
-                :cpu_submarine,
-                :player_cruiser,
-                :player_submarine
-
-    def initialize 
-        @player_board = Board.new
-        @cpu_board = Board.new
-        @cpu_cruiser = Ship.new("Cruiser", 3)
-        @cpu_submarine = Ship.new("Submarine", 2)
-        @player_cruiser = Ship.new("Cruiser", 3)
-        @player_submarine = Ship.new("Submarine", 2)
-    end
+    # attr_reader :player_board,
+    #             :cpu_board,
+    #             :cpu_cruiser,
+    #             :cpu_submarine,
+    #             :player_cruiser,
+    #             :player_submarine
 
-    def cpu_place_ships
+    # def initialize 
+    #     @player_board = Board.new
+    #     @cpu_board = Board.new
+    #     @cpu_cruiser = Ship.new("Cruiser", 3)
+    #     @cpu_submarine = Ship.new("Submarine", 2)
+    #     @player_cruiser = Ship.new("Cruiser", 3)
+    #     @player_submarine = Ship.new("Submarine", 2)
+    # end
 
-    end
+    # def cpu_place_ships
+
+    # end
     
-    def player_place_ships
+    # def player_place_ships
+        
+    # end
+end
+
+
+    def initialize
+      @player_board = Board.new
+      @cpu_board = Board.new
+    end
+  
+    def start_game
+            puts "Welcome to Battleship!"
         
+      
+      
+      place_player_ships
+  
+      
+      place_cpu_ships
+  
+            puts "All ships are placed. Game is ready to start!"
+    end
+  
+    def place_player_ships
+      
+        cruiser = Ship.new("Cruiser", 3)
+            puts "Place your Cruiser (3 units long)."
+        player_place_ship(cruiser)
+  
+     
+        submarine = Ship.new("Submarine", 2)
+            puts "Place your Submarine (2 units long)."
+        player_place_ship(submarine)
```

Comment:
I think that we can probably pull the ships' names and lengths from the ship objects.